src/navigation/navigation-events.js

```diff
@@ -31,7 +31,13 @@ export function navigateToUrl(obj) {
 
 	if (url.indexOf('#') === 0) {
 		window.location.hash = '#' + destination.anchor;
-	} else if (!isSamePath(destination.path, current.path) || (current.host !== destination.host && destination.host)) {
+	} else if (current.host !== destination.host && destination.host) {
```

Comment:
We need to handle cross domain navigation separately from same domain navigation